src/lib/storage/base_dictionary_column.hpp

```diff
@@ -1,27 +0,0 @@
-#pragma once
```

Comment:
Since this will change the interface, could you please take the time to sit down with the seminar groups that are using it? This is mostly the data types group, but the join and pruning groups might be affected too. Feel free to do this in your Hiwi time.